Tests/CustomDumpTests/DiffTests.swift

```diff
@@ -446,6 +446,29 @@ final class DiffTests: XCTestCase {
     )
   }
 
+  func testEnumCollapsing() {
```

Comment:
We didn't have any tests for collapsing enums, so I added one while troubleshooting the main issue.